esp/scm/ws_workunits.ecm

```diff
@@ -1471,9 +1471,11 @@ ESPresponse [exceptions_inline] WUQueryDetailsResponse
     [min_ver("1.46")] string CompileTime;
     [min_ver("1.46")] ESParray<string> LibrariesUsed;
     [min_ver("1.46")] int CountGraphs;
-    [min_ver("1.46")] ESParray<string> GraphIds;
+    [min_ver("1.46"), depr_ver("1.62")] ESParray<string> GraphIds;
```

Comment:
EclWatch does need to recode to use the new ones which give more graph information as in WsWorkunits.WUInfo. The https://track.hpccsystems.com/browse/HPCC-14837 is for recoding EclWatch.